pkg/client/clientset/versioned/clientset.go

```diff
@@ -1,7 +1,7 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
The whole generated clientset `pkg/client` should be removed, it's not generated anymore.

Reply:
great catch, I actually believed this was still used for the operator behind the scene.